### DISCUSSION TITLE
Are there plugins for read/write Linux ext4 partitions on doublecmd for Windows？

### DISCUSSION THREAD

**Discussion by jack-liew** [doublecmd/doublecmd#893] [Suggestions]
thanks

**Comment by JonnyTech** [comment]
Diskinternals Reader for TC: Universal disk reader for Total Commander

Works well with Double Commander. Note that it is read only.

https://www.diskinternals.com/reader-for-tc/

**Reply by jack-liew** [reply]
Thank U
